### PR TITLE
Window - fix language issues

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Video_Display.md [hunk 1]
```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
...
-and the window share the same Tile Data Table.
-
-Whether the window is displayed can be toggled using
-[LCDC bit 5](#lcdc-5-window-display-enable). Enabling the window makes
+### Window
+
+Besides the Background, there is also the Window overlaying the
+Background. The Window is not scrollable, that is, it is always
+displayed starting from its top left corner. The location of the Window
+on the screen can be adjusted via the WX and WY registers. The screen
```
Comment (ISSOtm) [inline content/Video_Display.md:571]: We might want to significantly reword this section to explain that the Window is a tilemap, and the only "configurable" parameter is the on-screen position of its top-left pixel? Or maybe it's explained somewhere else in the doc, in which case this should be removed?

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Video_Display.md [hunk 2]
```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
...
-for the window are stored in the Tile Data Table. Both the Background
-and the window share the same Tile Data Table.
-
-Whether the window is displayed can be toggled using
-[LCDC bit 5](#lcdc-5-window-display-enable). Enabling the window makes
+### Window
+
+Besides the Background, there is also the Window overlaying the
+Background. The Window is not scrollable, that is, it is always
+displayed starting from its top left corner. The location of the Window
```
[the remark this answers is not in the document]
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Rewritten

File: content/Video_Display.md [hunk 3]
```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
...
+on the screen can be adjusted via the WX and WY registers. The screen
+coordinates of the top left corner of the Window are WX-7,WY. The tiles
```
[the remark this answers is not in the document]
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

File: content/Video_Display.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Rewritten

**Review by ISSOtm — APPROVED** [review]
Nice!

File: content/Video_Display.md [hunk 4]
```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
...
-on the screen can be adjusted via WX and WY registers. Screen
-coordinates of the top left corner of a window are WX-7,WY. The tiles
-for the window are stored in the Tile Data Table. Both the Background
-and the window share the same Tile Data Table.
-
-Whether the window is displayed can be toggled using
-[LCDC bit 5](#lcdc-5-window-display-enable). Enabling the window makes
+### Window
+
+Besides the Background, there is also the Window overlaying the
```
Comment (avivace) [inline content/Video_Display.md:568]: "Besides the Background, there is also the Window overlaying the Background."
I don't think this sounds better

File: content/Video_Display.md [hunk 5]
```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
...
+Background. The content of the Window is not scrollable, that is, it is always
```
Comment (avivace) [inline content/Video_Display.md:569]: "that is" is unnecessary

File: content/Video_Display.md [hunk 6]
```diff
@@ -563,21 +563,21 @@ Whether the background is displayed can be toggled using
...
+coordinates of the top left corner of the Window are (WX-7,WY). The tiles
+for the Window are stored in the Tile Data Table. Both the Background
+and the Window share the same Tile Data Table.
+
+Whether the Window is displayed can be toggled using
+[LCDC bit 5](#lcdc-5-window-display-enable). Enabling the Window makes
 [Mode 3](#lcd-status-register) slightly longer on scanlines where it's visible.
 (See [above](#ff4a-wy-window-y-position-r-w-ff4b-wx-window-x-position-minus-7-r-w)
-for the definition of "when the window is visible".)
+for the definition of "when the Window is visible".)
```
Comment (avivace) [inline content/Video_Display.md:580]: "Window visibility" may sound better

**Review by avivace — CHANGES_REQUESTED** [review]

File: content/Video_Display.md (hunk 4 above)
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

File: content/Video_Display.md (hunk 5 above)
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

File: content/Video_Display.md (hunk 6 above)
Comment (AlvaroBurnett) [reply content/Video_Display.md]: Fixed

**Review by avivace — APPROVED** [review]

**Pull request merged**